### ISSUE TITLE
color contrast issue with Landmarks over target page

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#694]
## Describe the bug
Landmarks drawn over the target page from adhoc tools or Assessment, do not comply with color contrast.

**To Reproduce**

1. Go to https://www.washington.edu/accesscomputing/AU/after.html#
2. turn on Landmarks from adhoc tools or assessment
3. See color contrast issues:

- purple: Navigation
- blue: region 
- pink: main

## Expected behavior

No color contrast issues, text is easy to read.

## Screenshots

![Screenshot landmarks over target page](https://user-images.githubusercontent.com/9062104/58199516-7b249380-7c85-11e9-8ce7-02fb47b5a8ae.png)

## Extension (please complete the following information)

- OS: [e.g. Windows/Mac] Windows
- Version [e.g. 1.140.1] 2.4.1
- Browser Version [e.g Chrome 71.0] 74.0.3729.157
- Target Page [e.g www.bing.com] https://www.washington.edu/accesscomputing/AU/after.html#

## Are you willing to submit a PR?

If asked, will you be willing to submit a PR to fix this bug? No

## Did you search for similar existing issues?

Did you search for similar issues in our github issues or on [stackoverflow](https://stackoverflow.com/questions/tagged/accessibility-insights)
Yes

**Comment by ferBonnin** [comment]
Font size is 20px, based on [Techniques G145](https://www.w3.org/WAI/WCAG21/Techniques/general/G145): Point sizes are based on the CSS pt size CSS3 Values. The ratio between sizes in points and CSS pixels is 1pt = 1.333px, therefore 14pt and 18pt are equivalent to approximately 18.5px and 24px.

So two options:

- make font bigger
- change colors (need to consider high contrast mode too)

**Comment by lisli1** [comment]
@cheade the proposed solutions would change the UI. What are your thoughts?

**Comment by cheade** [comment]
Hi guys! I've played around with the CSS and I have two different options. (pt to px size is also determined by the DPI of one's screen). I had a third option where option 2 just has black text. Even though it passes color contrast, I didn't like it at all. 

### Option 1
Every landmark title gets a white background with black text, which I made 16px, as I feel it makes it a little easier to spot, as well as a dashed line around the text box with the corresponding color. This will keep color contrast from being an issue with the text both AA and AAA. 
![Group](https://user-images.githubusercontent.com/40274682/58210997-88e71280-7c9f-11e9-81e4-fb35d042a046.png)

### Option 2
Give the text box a dashed line in the same color, keep the color matching background, make text 14pt and bold. One of the problems I recall before were users not necessarily spotting the landmarks very well. A bigger header will make them much easier to spot for users!
![Group 2](https://user-images.githubusercontent.com/40274682/58211109-db283380-7c9f-11e9-808b-315f01536cef.png)

**Comment by ferBonnin** [comment]
@cheade thanks for the options! 
Question, I noticed you changed the font color too to white for some cases in opt 2, is that part of the option?

**Comment by cheade** [comment]
Yep! All of these colors should pass with 14pt bold on white.

![image](https://user-images.githubusercontent.com/40274682/58211425-be403000-7ca0-11e9-8e38-74c4d1b5c2cc.png)

**Comment by ferBonnin** [comment]
For summary, lets use opt 2. This includes changing the text color to white instead of black.
dashing the box around as shown in opt 2, is nice to have.

**Comment by Shobhit1** [comment]
Working on it.
